sample/android/build.gradle.kts

```
@@ -55,6 +55,9 @@ dependencies {
   implementation(libs.androidx.activity.compose)
   implementation(libs.androidx.profileinstaller)
 
+  implementation(libs.androidx.compose.ui.tooling)
+  implementation(libs.androidx.compose.ui.preview)
```

According to [the documentation](https://developer.android.com/develop/ui/compose/setup#setup-compose), the `ui-tooling` artefact should be `debugImplementation`:
```suggestion
  debugImplementation(libs.androidx.compose.ui.tooling)
  implementation(libs.androidx.compose.ui.preview)
```